cmd/root.go

```diff
@@ -52,7 +52,11 @@ There will be a ClusterPolicyReport with results for cluster-wide resources.`,
 			}
 			scanner := scanner.NewScanner(policiesFetcher, resourcesFetcher)
 
-			return startScanner(namespace, scanner)
+			err = startScanner(namespace, scanner)
+			if err != nil {
+				return err
+			}
+			return nil
```

Comment:
I think the original code was better (and idiomatic), this is just a more verbose way to do the very same thing.

Reply:
was this about `Execute()`? I can't see any change on `startScanner()`. Maybe I'm missing something. Just in case, left `Execute()` back how it was.

Reply:
I mean that `startScanner` is already returning an `error`, hence it's feels more natural to do:

```golang
return startStanner(namespace, scanner)
```

rather than:

```golang
err = startScanner(namespace, scanner)
if err != nil {
  return err
}
return nil
```

Reply:
oh, indeed, done in https://github.com/kubewarden/audit-scanner/commit/5fc30be0cad27129eda6b3c4a38e10a336d5cc22.